convert2rhel/actions/system_checks/eus.py

```diff
@@ -0,0 +1,52 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import datetime
+import logging
+
+from convert2rhel import actions
+from convert2rhel.systeminfo import EUS_MINOR_VERSIONS, system_info
+from convert2rhel.toolopts import tool_opts
+
+
+logger = logging.getLogger(__name__)
+
+
+class EusSystemCheck(actions.Action):
+    id = "EUS_SYSTEM_CHECK"
+
+    def run(self):
+        """Inhibit the conversion when we are not able to handle UEFI."""
+        super(EusSystemCheck, self).run()
+
+        current_version = "%s.%s" % (system_info.version.major, system_info.version.minor)
+        if current_version == "8.8":
+            eus_release_date = EUS_MINOR_VERSIONS.get(current_version, False)
+            # Turn eus_release_date into a datetime object
+            eus_release_date = datetime.datetime.strptime(eus_release_date, "%Y-%m-%d")
```

Comment:
```suggestion
            eus_release_date = datetime.datetime.strptime(eus_release_date, "%Y-%m-%d").date()
```
This needs to be converted to the `date` object, to be [comparable with the current_datetime here.](https://github.com/oamg/convert2rhel/pull/947/files#diff-1470230e5916d6ab112f1569ea65dc18608f566182e2c0ae0a33cbe0b2a0e7a4R44)